docs/src/css/custom.css

```diff
@@ -0,0 +1,69 @@
+/* stylelint-disable docusaurus/copyright-header */
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
+
+/* You can override the default Infima variables here. */
+
+@import url('https://fonts.googleapis.com/css2?family=Rubik');
+
+:root {
+  --ifm-color-primary: #25c2a0;
+  --ifm-color-primary-dark: #409088;
+  --ifm-color-primary-darker: #387462;
+  --ifm-color-primary-darkest: #1b4e3f;
+  --ifm-color-primary-light: #42ba96;
+  --ifm-color-primary-lighter: #86b8b6;
+  --ifm-color-primary-lightest: #abd5c6;
+  --ifm-code-font-size: 95%;
+  --ifm-spacing-horizontal: 1em;
+  --ifm-font-family-base: "Rubik", system-ui, -apple-system, Segoe UI, Roboto, Ubuntu, Cantarell, Noto Sans, sans-serif, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
```

Comment:
Does our style guide offer any different typefaces for multi-paragraph text? Rubik looks great in small chunks on solana.com, but I find it uncomfortable to read in the context of the docs, especially in dark mode.

Reply:
@CriesofCarrots let me ask Raj and see what we can do. I agree with this completely. Maybe we can keep Roboto.

Reply:
@CriesofCarrots let me know if you are a fan of Roboto!

Reply:
Yes, I am!